erpnext_documentation/www/docs/user/manual/en/stock/opening-stock.md

```diff
@@ -10,32 +10,18 @@ The closing Stock of the previous accounting period becomes the opening Stock of
 * Create [Warehouses](/docs/user/manual/en/stock/warehouse).
 * Link Warehouse to appropriate accounting ledgers.
 
-## 2. Opening Stock for Non-serialized Items
-
+## 2. Opening Stock for Items
 To post opening stock visit the [Stock Reconciliation](/docs/user/manual/en/stock/stock-reconciliation) page.
 
-
-## 3. Opening Stock for Serialized and Batched Items
```

Comment:
why remove, this is still an alternate method?

Reply:
@prasadarr while it is an alternative method, the features have already been added in Stock Reco.

Perhaps then we'll need to re-write certain sections:

1. Opening stock using Stock Entry

2. Opening stock using Stock Reco

Reply:
@prasadarr thoughts?

Reply:
yeah that's nice, 
1. via Stock Reco. add a note saying this method is recommended.
2. via Stock Entry. add a note saying this is an alternate older method.